### PR TITLE
nodejs-ca: Add get-involved file - index.md

### PR DESCRIPTION
**Pull request by Tsuna-mi**
This PR is related to [#1197](https://github.com/nodejs/nodejs.org/issues/1197)

I have created the '**get-involved**' index.md translation to catalan, according Node.js Website Translation Policy indications.

nodejs-ca: Add get-involved file - index.md

### CODE REVIEW

File: locale/ca/get-involved/index.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+- [NodeUp](http://nodeup.com) és un podcast cobrint les últimes notícies en la comunitat de Node.
+
+- [NodeJS Reactions](http://nodejsreactions.tumblr.com) captura l'experiència de Node.js en forma de GIFs animats.
+
+
+## Aprenentatge
+
+- La [Documentació oficial de l'API](/api) detalla l'API de Node.
+
+- [NodeSchool.io](http://nodeschool.io) li ensenyarà conceptes de Node.js de forma interactiva mitjançant jocs utilitzant la línia de comandos.
```
Comment (joselaradelamo) [inline locale/ca/get-involved/index.md]: comandes*

File: locale/ca/get-involved/index.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+# Fomentem tota classe de contribucions de la comunitat.
+
+La comunitat de Node.js és gran, inclusiva i entusiasmada per permetre al major nombre d'usuaris,
+contribuir de la manera que puguin. Si vols [reportar un error](https://github.com/nodejs/node/issues),
+[ajudar amb la documentació](/en/get-involved/contribute/) o [contribuir amb el codi](/en/get-involved/development/) del projecte, has vingut al lloc indicat. Explora els recursos de la nostra comunitat per trobar una forma d'ajudar:
+
+<hr>
+
+## Discusió
```
Comment (jose-lara) [inline locale/ca/get-involved/index.md]: Discussió*

File: locale/ca/get-involved/index.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+- [NodeSchool.io](http://nodeschool.io) li ensenyarà conceptes de Node.js de forma interactiva mitjançant jocs utilitzant la línia de comandos.
+
+- L'[etiqueta de Node.js en StackOverflow](http://stackoverflow.com/questions/tagged/node.js) col·lecciona nova informació cada dia.
+
+- [docs.nodejitsu.com](http://docs.nodejitsu.com/) soluciona molts dels problemes més comuns que la gent es troba.
+
+- [How To Node](http://howtonode.org/) té un nombre creixent d'útils tutorials.
+
+
+## Llocs de la comunitat internaciona i projectes
```
Comment (jose-lara) [inline locale/ca/get-involved/index.md]: internacional*

Comment (Tsuna-mi) [reply locale/ca/get-involved/index.md]: All changes applied. Thanks a lot for the review @jose-lara :D

**Review by joselaradelamo — APPROVED** [review]

**Comment by fhemberger** [comment]
Great, first one landed!

**Pull request merged**